proto/lbm/foundation/v1/foundation.proto

```diff
@@ -0,0 +1,280 @@
+syntax = "proto3";
+package lbm.foundation.v1;
+
+option go_package            = "github.com/line/lbm-sdk/x/foundation";
+option (gogoproto.equal_all) = true;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/duration.proto";
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/any.proto";
+import "cosmos_proto/cosmos.proto";
+
+// Params defines the parameters for the foundation module.
+message Params {
+  bool   enabled        = 1;
+  string foundation_tax = 2
+      [(gogoproto.customtype) = "github.com/line/lbm-sdk/types.Dec", (gogoproto.nullable) = false];
+}
+
+// ValidatorAuth defines authorization info of a validator.
+message ValidatorAuth {
+  string operator_address = 1;
+  bool   creation_allowed = 2;
+}
+
+// UpdateFoundationParamsProposal details a proposal to update params of cosortium module.
+message UpdateFoundationParamsProposal {
+  option (gogoproto.equal)            = false;
+  option (gogoproto.goproto_getters)  = false;
+  option (gogoproto.goproto_stringer) = false;
+
+  string title       = 1;
+  string description = 2;
+  Params params      = 3;
+}
+
+// UpdateValidatorAuthsProposal details a proposal to update validator auths on foundation.
+message UpdateValidatorAuthsProposal {
+  option (gogoproto.equal)            = false;
+  option (gogoproto.goproto_getters)  = false;
+  option (gogoproto.goproto_stringer) = false;
+
+  string                 title       = 1;
+  string                 description = 2;
+  repeated ValidatorAuth auths       = 3 [(gogoproto.nullable) = false];
+}
+
+// Member represents a foundation member with an account address,
+// non-zero weight and metadata.
+message Member {
+  // address is the member's account address.
+  string address = 1;
+
+  // weight is the member's voting weight that should be greater than 0.
+  string weight = 2 [(gogoproto.nullable) = false, (gogoproto.customtype) = "github.com/line/lbm-sdk/types.Dec"];
```

Comment:
Why does this weight need? I heard all member has same voting power.

Reply:
The planning team decided to keep the current design. So I forced the value to be 0 or 1, which is included in the corresponding `ValidateBasic()`.